test/models/session_test.exs

```diff
@@ -6,6 +6,27 @@ defmodule Yggdrasil.SessionTest do
 
   @user %User{:username => "tester"}
 
+  @correct_username "tester"
+  @correct_password "password"
+
+  @incorrect_username "zzzzzzz"
+  @incorrect_password "notpassword"
+
+  defp setup_user() do
+    user = %{
+      :username => @correct_username,
+      :password => @correct_password,
+      :password_confirmation => @correct_password
+    }
+
+    User.create_changeset(%User{}, user)
+      |> Repo.insert()
```

Comment:
So does ecto do a rollback after each test? Or are we populating the database and this will fail when you run it two times in a row?